pkg/kubelet/kubelet.go

```diff
@@ -2845,6 +2845,10 @@ func (kl *Kubelet) validateContainerLogStatus(podName string, podStatus *api.Pod
 	var cID string
 
 	cStatus, found := api.GetContainerStatus(podStatus.ContainerStatuses, containerName)
+	// if not found, check the init containers
```

Comment:
@lukaszo Trying to understand, this covers only the case where the container name is specified in the command line. right? (and not the problems mentioned in #27040)